### PR TITLE
feat(route): 添加Lovelive官网Schedule页的路由

### PR DESCRIPTION
**Pull request by axojhf**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/lovelive-anime/schedules
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/lovelive-anime/schedules.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    const items = response.data.data.schedule_list.map((item) => {
+        const link = item.url.select_url;
+        const title = item.title;
+        const category = item.categories.name;
+        const eventStartDate = dayjs.unix(item.event_startdate).tz('Asia/Tokyo').format('YYYY-MM-DD HH:mm:ss JST');
+        const eventEndDate = dayjs.unix(item.event_enddate).tz('Asia/Tokyo').format('YYYY-MM-DD HH:mm:ss JST');
+
+        return {
+            link,
+            pubDate: '',
```
Comment (TonyRL) [inline lib/v2/lovelive-anime/schedules.js:37]: Maybe set `pubDate` to eventStartDate?

Comment (axojhf) [reply lib/v2/lovelive-anime/schedules.js:37]: The `eventStartDate` in the website api is not recommended for `pubDate`, the data obtained by the API has more complex time attributes, so I simply leave the pubDate empty.

Comment (TonyRL) [reply lib/v2/lovelive-anime/schedules.js:37]: The data from the API is not complex at all. They are simply unix timestamps.
`pubDate: parseDate(item.created, 'X')` will do the trick. `created` is more preferred.
```json
{
  "event_enddate": 1668956340,
  "reset": 1663223829626,
  "updated_user": "脇黒丸 翔太",
  "lng": "jp",
  "startdate": 1662044400,
  "_p_key": "lovelive_jp",
  "site": "jp",
  "url": {
    "select_transition": "2",
    "select_url": "https://www.lovelive-anime.jp/yuigaoka/live/sp_3rdlive/ticket.php"
  },
  "series": [
    {
      "name": "スーパースター!!",
      "code": "superstar"
    }
  ],
  "_id": "13",
  "id": "13",
  "created": 1662094082,
  "ip": "lovelive",
  "event_startdate": 1664334000,
  "start_endflg": "1",
  "created_user": "脇黒丸 翔太",
  "updated": 1663223829,
  "enddate": 32503647540,
  "publish_status": "publish",
  "categories": {
    "name": "ライブ",
    "code": "live"
  },
  "delflg": "0",
  "event_dspdate": "＜最速先行抽選＞申込受付期間：2022年9月28日～2022年11月20日（TVアニメ2期 Blu-ray第1巻封入）",
  "period_label": "受付中",
  "title": "【チケット情報】ラブライブ！スーパースター!! Liella! 3rd LoveLive! Tour ～WE WILL!!～＜千葉公演＞"
}
```

Comment (axojhf) [reply lib/v2/lovelive-anime/schedules.js]: I just discovered the json has a key called `start_endflg`, it identifies whether to use `event_startdate` or `event_enddate`.
I have committed the code related to `pubDate`

**Pull request merged**